python/graphstorm/run/gsgnn_lp/gsgnn_lp.py

```diff
@@ -48,7 +48,7 @@
 from graphstorm.eval import GSgnnMrrLPEvaluator, GSgnnPerEtypeMrrLPEvaluator
```

Comment:
We also need to change lp_infer_gnn.py, ep_infer_gnn.py and np_infer_gnn.py

Reply:
Moved to another PR.